tensorrt_llm/_torch/auto_deploy/transform/library/fused_moe.py

```
@@ -1513,6 +1547,8 @@ def _prepare_args_triton_format():
 
         node.replace_all_uses_with(new_node)
         graph.erase_node(node)
+        eliminate_dead_code(gm)
```

Ruff (0.14.13)

Undefined name `eliminate_dead_code`

(F821)